spec/controllers/ballots_controller_spec.rb

```diff
@@ -21,19 +21,22 @@
 
   describe "POST #create" do
     subject { post :create, { beer_id: beer.id } }
-    let!(:poll) { FactoryGirl.create :poll }
+    let(:poll) { FactoryGirl.create :poll }
 
     it "creates a new ballot" do
+      poll
```

Comment:
Instead of writing this three times, it would be better to keep `let!` and wrap all of these expectations in a context:

```ruby
context "when an open poll exists" do
```